### DISCUSSION TITLE
Run multiple ql files and create a path-problem report

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#8492] [General]
I'm trying to run all QL files in `vscode-codeql-starter/ql/java/ql/src/Security/CWE` that is a `@kind path-problem`. How can I do that?

1. Not sure how to do that in cli
2. VSCode allow me to run 1 QL file at a time

**Comment by smowton** [comment]
You could use the `codeql database analyze` command, passing a `.qls` file identifying the queries you want.

The command: https://codeql.github.com/docs/codeql-cli/manual/database-analyze/

QLS files (query suites): https://codeql.github.com/docs/codeql-cli/creating-codeql-query-suites/

Example query suite: https://github.com/github/codeql/blob/main/java/ql/src/codeql-suites/java-code-scanning.qls

Which references: https://github.com/github/codeql/blob/main/misc/suite-helpers/code-scanning-selectors.yml
